machine/firecracker/v1alpha1.go

```diff
@@ -87,8 +87,8 @@ func (service *machineV1alpha1Service) Create(ctx context.Context, machine *mach
 		return machine, fmt.Errorf("cannot create firecracker instance with emulation")
 	}
 
-	if !cpuid.CPU.Rdrand() || !cpuid.CPU.Rdseed() {
-		log.G(ctx).Warn("RDRAND and RDSEED are not supported by the host CPU to be able to run Unikraft v0.17.0 and greater with hardware randomization")
+	if !cpuid.CPU.Has(cpuid.RDRAND) {
+		log.G(ctx).Warn("RDRAND is not supported by the host CPU to be able to run Unikraft v0.17.0 and greater with hardware randomization")
```

Comment:
Since you're updating this line, the term normally used is "randomness", not "randomization". Or if you want to be more precise, "CPU-generated randomness".